nvflare/lighter/impl/master_template.yml

```diff
@@ -1863,8 +1662,8 @@ aws_start_cln_sh: |
     echo "Launching container with docker option ${DOCKER_OPTION}."
     ssh -f -i $KEY_FILE -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null ${DEST_SITE} \
     "docker run -d -v ${DEST_FOLDER}:${DEST_FOLDER} --network host ${DOCKER_OPTION} ${image_name} \
-    /bin/bash -c \"python -u -m nvflare.private.fed.app.client.client_train -m ${DEST_FOLDER} \
-    -s fed_client.json --set uid={~~SITE~~} secure_train=true config_folder=config org={~~ORG~~} \" " > /tmp/nvflare.log 2>&1 
+    /bin/bash -c \"python -u -m nvflare.private.fed.app.{~~type~~}.{~~type~~}_train -m ${DEST_FOLDER} \
+    -s fed_{~~type~~}.json --set {~~cln_uid~~} secure_train=true config_folder=config org={~~ORG~~} \" " > /tmp/nvflare.log 2>&1 
```

Comment:
is this supposed to be `--set uid={~~cln_uid~~}` or do we not need the `uid=`?

Reply:
Originally, we have uid={~~cln_uid~~} and {~~cln_uid~~} will be replaced with the site name.  Now we replace {~~cln_uid~~} with uid=site1 (if the site name is site1).  This makes the results the same.